### ISSUE TITLE
Bill of Materials

### ISSUE THREAD

**Issue by DanielArnett** [Roboy/HTC_Vive_Sensors#1]
Is there a bill of materials for assembling a photodiode board? I see the part numbers but when I look them up (for example [the photodiode](https://www.digikey.com/products/en/sensors-transducers/optical-sensors-photodiodes/543?mnonly=0&ColumnSort=0&page=1&stock=0&pbfree=0&rohs=0&cad=0&datasheet=0&nstock=0&photo=0&nonrohs=0&newproducts=0&k=BPW34&quantity=&ptm=0&fid=0&pageSize=50) ) there are many different sizes and form factors to choose from.

**Comment by letrend** [comment]
We ordered the big parts from mouser:
https://www.mouser.de/ProjectManager/ProjectDetail.aspx?AccessID=4f7b87558e

**Comment by DanielArnett** [comment]
Great! Thank you. 

[Here's a BOM](https://docs.google.com/spreadsheets/d/1M4g7Zi2gSui11CSCxC3s6bzpGJjz4p6uyUDvzPWtF8c/edit?usp=sharing) for a single board, with the links you've provided. Can you check one or two of the resistors/capacitors I selected? Like [C1](https://www.mouser.com/ProductDetail/AVX/04023C104KAT2A/?qs=sGAEpiMZZMsh%252b1woXyUXj4WHJD67K%2fZZLVDWJLjP28I%3d) and [R1](https://www.mouser.com/ProductDetail/Vishay-Beyschlag/MCS0402MD1002BE100/?qs=sGAEpiMZZMtlubZbdhIBIPT49CK2htFMtrrJJVOhDGo%3d)? If those look correct then the rest of them probably are too. 

Here's the [Mouser Cart](https://www.mouser.com/ProjectManager/ProjectDetail.aspx?AccessID=e63a7fa8c1) of components needed for 30 boards.

[Also here's a link to purchase the board as is.](https://www.pcbway.com/project/shareproject/W53271ASI1_gerb_smd.html)

**Comment by letrend** [comment]
You're more missing the adjusted gain resistors. Have a look at this again: https://github.com/Roboy/HTC_Vive_Sensors/blob/master/V0.3/Vive_sensors_smd.ods

The two 100k in parallel equal 50k. But those are quite expensive. Instead you can use one 100k and one 2k resistor for achieving the same gain.

Nice one on pcbway! I can really recommend the 4x4 panel, which speeds up production a lot.

**Comment by DanielArnett** [comment]
Oh wow I didn't even see that BOM. I don't quite follow what you mean by Current gain. Why are there multiple R7,R8,R9, and R12 resistors?

I have the boards ordered, the FPGA dev board, IMU, and wifi board have arrived. Hopefully next week we'll have the boards populated and upload the FPGA code.

Also [here's an idea of what we're doing with our project](https://drive.google.com/file/d/0B7YXimF4OJdJM2E4MHdPOWZ1dDQ/view?usp=sharing) (we're using the commercial Vive Tracker at the moment).

**Comment by letrend** [comment]
The resistors on the bottom have been found to work well for our use case, ie minimum of 1.5m from each lighthouse and about 5 meter maximal distance. They set the amplification gains of the opamps. 
So as I said, instead of the 2 parallel 100k resistors (we did this because we had no 50k resistors), use one 100k and instead of the 1k use 2k. The gain 50k/1k is the same as 100k/2k.

Which wifi board did you order?

Wow, I have difficulty judging the size of you rovers. Is that a lighthouse mounted on the left one?

**Comment by DanielArnett** [comment]
Yes, lighthouse base station on a tripod on the right. The rovers are
almost a meter long which actually brings up a relevant point. If I wanted
to track something closer than 1.5m, say 10 centimeters from the base
station, do you think it would be a matter of adjusting the amplification
gains?

**Comment by DanielArnett** [comment]
[We got this wifi board.](https://www.adafruit.com/product/2471) from the hackaday.

**Comment by letrend** [comment]
It will always be a compromise. You can reduce the gains in  the circuit, such that the signal can be decoded 10cm in front of a lighthouse. But the maximal distance will then also be reduced accordingly. So the rovers are 1m long already... What exactly are you trying to track 10 cm in front of a lighthouse?

**Comment by DanielArnett** [comment]
That's great! What about a mixed system? Some photodiodes have their gain tuned for 1.5-5.0m, some  have their gain appropriate for 0.5-2.0m. For my project I would really like to be able to use the tracker at short range so I have more flexibility on where I place the Base stations on my rover.

**Comment by DanielArnett** [comment]
Okay can you recheck our updated BOM?

[Here's the BOM as a Mouser Order.](https://www2.mouser.com/ProjectManager/ProjectDetail.aspx?State=EDIT&ProjectGUID=f82a3a47-e2cb-479f-a208-2fe63a7fa8c1)

[Here's the BOM in Spreadsheet form.](https://docs.google.com/spreadsheets/d/1M4g7Zi2gSui11CSCxC3s6bzpGJjz4p6uyUDvzPWtF8c/edit?usp=sharing)

**Comment by letrend** [comment]
Looks alright

**Comment by DanielArnett** [comment]
So what resistors do I use for R7, R8, R9, and R12 (the gain resistors)? Should I just not use the originals and use the ones listed under gain instead? (Board fabrication started today)

**Comment by letrend** [comment]
Yes, use the ones listed under gains
